assets/stylesheets/directly.scss

```diff
@@ -0,0 +1,503 @@
+
+	&::-webkit-input-placeholder {
+		color: $gray;
+	}
+
+}
+textarea {
+	vertical-align: inherit;
+}
+
+.minimized-view,
+.maximized-view {
+	background: #F2F6F8;
+}
+.minimized-view {
+	background: $white;
+	box-shadow: 0 1px 10px rgba(46,68,83,.1);
+	border: 1px solid lighten( $gray, 25% );
+}
+.maximized-view.is-ask-view {
+	top: 0 !important;
+}
+.message-list-transition-wrapper {
+	background: #F2F6F8;
+}
+
+
+/* HEADER */
+
+.header-container {
+	border-bottom: 1px solid lighten( $gray, 30% );
+	box-shadow: none;
+	flex: 0 0 32px;
+	height: 32px;
+
+	.header-title {
+		text-transform: uppercase;
+		font-size: 11px;
+		font-weight: 700;
+		color: $gray;
+	}
+
+	// Left hamburger button
+	.menu {
+		border-right: none;
+		flex: 0 0 40px;
+		.svg-icon {
+			color: $gray;
+		}
+		&.disabled {
+			background: transparent;
+			.svg-icon {
+				color: lighten( $gray, 25% );
+			}
+		}
+	}
+
+	// Right minimize button
+	.minimize-container {
+		border-left: none;
+		flex: 0 0 40px;
+		.svg-icon {
+			color: $gray;
+		}
+	}
+
+	.thread-view-title-container {
+		&.has-notification {
+			border-bottom-color: #14abdd;
+		}
+	}
+}
+
+
+/* CHAT LIST (opens with the hamburguer button on header) */
+
+.chat-list-view-container {
+	.chat-list-container {
+		top: 32px;
+		.question-bar,
+		.question-bar.closed {
+			background: $gray;
+			padding: 10px 14px;
+		}
+	}
+}
+
+.chat-list-view-item-container {
+	background: lighten( $gray, 30% );
+	border-bottom: 1px solid lighten( $gray, 25% );
+	color: $gray-dark;
+	padding: 18px 10px 18px 24px;
+	.chat-list-item-title {
+		margin-bottom: 2px;
+		font-size: 16px;
+	}
+	.chat-list-item-meta {
+		color: $gray;
+	}
+	.unread-message-dot {
+		background: $alert-green;
+		left: 10px;
+		top: 25px;
+	}
+}
+
+// Bubble when the chat list is minimized
+.chat-list-view-minimized {
+	.chat-list-status {
+		margin-top: 2px;
+	}
+	.svg-icon {
+		color: $blue-medium;
+	}
+}
+
+
+/* ASK QUESTION (opens from chat list) */
+
+.ask-question-form-container {
+	.input-group {
+		padding-bottom: 12px;
+		margin-top: 0;
+		label {
+			text-transform: none;
+			font-size: 14px;
+			font-weight: 600;
+			margin-bottom: 5px;
+			strong {
+				font-weight: inherit;
+			}
+		}
+	}
+	.submit {
+		width: 100%;
+		padding-bottom: 9px;
+		.svg-icon {
+			display: none;
+		}
+		strong {
+			margin-top: 0;
+			font-weight: 500;
+		}
+	}
+}
+
+
+/* THREAD */
+
+.thread-view-container {
+	.expert-response-time {
+		color: $gray;
+	}
+}
+
+// Message bubbles
+.message-container {
+	font-size: 14px;
+	border-radius: 8px;
+
+	&.is-poster {
+		background: #14abdd;
+		margin-left: inherit;
+		margin-right: 20px;
+		border-top-right-radius: 0px;
+		position: relative;
+		&:after {
+			// Triangle for the corner
+			position: absolute;
+			content: "";
+			display: block;
+			width: 0;
+			height: 0;
+			border-style: solid;
+			border-width: 10px 10px 0 0;
+			border-color: #14abdd transparent transparent transparent;
+			top: 0;
+			right: -10px;
```

Comment:
Yep you're right! https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#positioning

I'll make that change for the places where `position` is explicitly used.